### DISCUSSION TITLE
Filter chart by date between two time columns

### DISCUSSION THREAD

**Discussion by hrvylein** [apache/superset#25233] [Q&A / Help]
I'm desperately trying to set up a (cross) filter that filters dashboards by comparing if the chosen date is between two time columns in the query.

From my understanding I can't precompute anything, as the filter date drives the data.

I tried with jinja twmplating but this doesn not seem to work, as my cross filter isn't detected.

How can I achieve this behaviour? On the other hand there are a a lot of charts in this dashboard where selected date is only one column.

Can you help me out or give me a hint in the right direction?

Thanks!

**Comment by A-Kugler** [comment]
Is there any solution to this, please? @hrvylein

**Reply by hrvylein** [reply]
I never got it working
